LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright for portions of naoufal/react-native-payments are held by Truphone as part of project truphone/react-native-payments, 2018. All other copyright for react-native-payments are held by the upstream mainainer (naoufal, 2016-2018) and the contributors listed in https://github.com/Truphone/react-native-payments/graphs/contributors.
```

Comment:
This seems wrong to include in this PR!

Reply:
@fbartho you're right, i reverted the license change back to original